src/transformers/models/canine/modeling_canine.py

```diff
@@ -1467,9 +1469,64 @@ def __init__(self, config):
     @add_start_docstrings_to_model_forward(CANINE_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="aliosm/sha3bor-poetry-diacritizer-canine-s",
         output_type=TokenClassifierOutput,
         config_class=_CONFIG_FOR_DOC,
+        expected_output=[
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_2",
+            "LABEL_2",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_0",
+            "LABEL_1",
+            "LABEL_0",
+            "LABEL_3",
+            "LABEL_3",
+        ],
+        expected_loss=0.46,
```

Comment:
This list is too long and doesn't really provide meaningful information, as the model is character-based. If the list is short, we could accept it (as done in a few other doc examples).